### ISSUE TITLE
Swagger spec incorrect casing for complex type parameters

### ISSUE THREAD

**Issue by Snelbinder** [domaindrivendev/Swashbuckle#322]
I'm using camel-casing for my route parameters, sometimes I use complex types as parameter which works fine. However, no matter what casing I use for the names of the properties, they are all in lower case in the generated Swagger spec.

In the XML the member looks as follows:
 ```
        <member name="NavigationFlagParameters.IncludeChildNodes">
            <summary>
            Indicates whether or not child nodes should be included. 
            </summary>
        </member>
```
This will be converted into the following in the spec:
```
{
    "name": "includechildnodes",
    "in": "query",
    "description": "Indicates whether or not child nodes should be included.",
    "required": false,
    "type": "boolean"
},
```
Am I missing something or is this a bug?

**Comment by domaindrivendev** [comment]
Yep this looks like a bug. Thanks for reporting it - I'll aim to get a fix out with the next release.

**Comment by domaindrivendev** [comment]
Should be resolved in latest release - 5.2.0

**Comment by mcoolidge** [comment]
It seems like all my request parameters are FORCED to camelCase. This looks wierd when I have things like DOB or SSN which become sSN and dOB
